project/sbt-updates.sbt

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
```

Comment:
I normally prefer to define this plugin (as well as other convenience plugins, like dependency tree) globally instead of doing it in every project. But it's just a nitpick.